### PR TITLE
Fix hidden nested images in content column

### LINKED ISSUE #18296: Media/Text block bug with nested Image block: Image is not visible
Hey,

if I use the media/text block with the activated option "`Crop image to fill entire column`" and a nested image inside the text column, the image is still hidden.

The backend view:

Bildschirmfoto 2019-11-06 um 09 11 15

The frontend view:

Nov-06-2019 09-13-50

This default block styles affect this design issue and is the reason, why the nested image is still hidden:

```
.wp-block-media-text.is-image-fill figure>img {
    position: absolute;
    width: 1px;
    height: 1px;
    padding: 0;
    margin: -1px;
    overflow: hidden;
    clip: rect(0,0,0,0);
    border: 0;
}
```

Please note that this problem only occurs when the option "`Crop image to fill entire column`" is activated.

### PR DESCRIPTION
**Pull request by CreativeDive**
Media/Text block bug with nested Image block: Image is not visible 

closes #18296

### CODE REVIEW

File: packages/block-editor/src/components/block-list/style.scss [hunk 1]
```diff
@@ -448,7 +448,7 @@
 	// Left
 	&[data-align="left"] {
 		// This is in the editor only; the image should be floated on the frontend.
-		.block-editor-block-list__block-edit {
+		> .block-editor-block-list__block-edit {
```
Comment (youknowriad) [inline packages/block-editor/src/components/block-list/style.scss]: Why do we need this change? I think the changes in the other files fix the issue for me.

Comment (CreativeDive) [reply packages/block-editor/src/components/block-list/style.scss]: @youknowriad That's right! Something goes wrong with the pull request. I don't know why the files from an older pull request are included too. 

The changes in "packages/block-library/src/media-text/style.scss" fix this issue.

Comment (youknowriad) [reply packages/block-editor/src/components/block-list/style.scss]: Happy to merge the PR if you can manage to clean it :)

Comment (CreativeDive) [reply packages/block-editor/src/components/block-list/style.scss]: Not sure how I can edit this pull request to remove the part "e0d6636"?

Comment (youknowriad) [inline packages/block-editor/src/components/block-list/style.scss]:
```suggestion
		.block-editor-block-list__block-edit {
```

Comment (youknowriad) [reply packages/block-editor/src/components/block-list/style.scss]: You can just click the "commit suggestion" here. I can't do it myself as I don't have rights on your repository.

**Review by youknowriad — APPROVED** [review]
👍 Thanks

**Comment by CreativeDive** [comment]
> 👍 Thanks

Thank you for helping. A question for the future. How I can sync my fork with the current gutenberg master, before I edit code for the PR?

**Comment by youknowriad** [comment]
@CreativeDive This page might help https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork 

If you're still having trouble, let me know.

**Pull request merged**